src/libraries/System.Runtime.Extensions/tests/System/MathF.cs

```diff
@@ -1421,6 +1421,84 @@ public static void Pow_IEEE(float x, float y, float expectedResult, float allowe
+        [InlineData(-2.71828183f,            -0.36787945f,            CrossPlatformMachineEpsilon * 1000)]  // value: (e)
+        [InlineData(-2.30258509f,            -0.434294462f,           CrossPlatformMachineEpsilon * 1000)]  // value: (ln(10))
+        [InlineData(-1.57079633f,            -0.636619747f,           CrossPlatformMachineEpsilon * 1000)]  // value: (pi / 2)
+        [InlineData(-1.44269504f,            -0.693147182f,           CrossPlatformMachineEpsilon * 1000)]  // value: (log2(e))
+        [InlineData(-1.41421356f,            -0.707106769f,           CrossPlatformMachineEpsilon * 1000)]  // value: (sqrt(2))
+        [InlineData(-1.12837917f,            -0.886226892f,           CrossPlatformMachineEpsilon * 1000)]  // value: (2 / sqrt(pi))
+        [InlineData(-1.0f,                   -1.0f,                   CrossPlatformMachineEpsilon * 10000)]
+        [InlineData(-0.785398163f,           -1.27323949f,            CrossPlatformMachineEpsilon * 10000)] // value: (pi / 4)
+        [InlineData(-0.707106781f,           -1.41421354f,            CrossPlatformMachineEpsilon * 10000)] // value: (1 / sqrt(2))
+        [InlineData(-0.693147181f,           -1.44269502f,            CrossPlatformMachineEpsilon * 10000)] // value: (ln(2))
+        [InlineData(-0.636619772f,           -1.57079637f,            CrossPlatformMachineEpsilon * 10000)] // value: (2 / pi)
+        [InlineData(-0.434294482f,           -2.30258512f,            CrossPlatformMachineEpsilon * 10000)] // value: (log10(e))
+        [InlineData(-0.318309886f,           -3.14159274f,            CrossPlatformMachineEpsilon * 10000)] // value: (1 / pi)
+        [InlineData(-0.0f,                    float.NegativeInfinity, 0.0f)]
+        [InlineData( float.NaN,               float.NaN,              0.0f)]
+        [InlineData( 0.0f,                    float.PositiveInfinity, 0.0f)]
+        [InlineData( 0.318309886f,            3.14159274f,            CrossPlatformMachineEpsilon * 10000)] // value: (1 / pi)
+        [InlineData( 0.434294482f,            2.30258512f,            CrossPlatformMachineEpsilon * 10000)] // value: (log10(e))
+        [InlineData( 0.636619772f,            1.57079637f,            CrossPlatformMachineEpsilon * 10000)] // value: (2 / pi)
+        [InlineData( 0.693147181f,            1.44269502f,            CrossPlatformMachineEpsilon * 10000)] // value: (ln(2))
+        [InlineData( 0.707106781f,            1.41421354f,            CrossPlatformMachineEpsilon * 10000)] // value: (1 / sqrt(2))
+        [InlineData( 0.785398163f,            1.27323949f,            CrossPlatformMachineEpsilon * 10000)] // value: (pi / 4)
+        [InlineData( 1.0f,                    1.0f,                   CrossPlatformMachineEpsilon * 10000)]
+        [InlineData( 1.12837917f,             0.886226892f,           CrossPlatformMachineEpsilon * 1000)]  // value: (2 / sqrt(pi))
```

Comment:
This was the "worst" case of difference that was measured. 

```
1 / 1.12837917f                  == 0.886226892f
ReciprocalEstimate(1.12837917f ) == 0.886352539f
```

This gives a difference of ~`0.000125647` in expected vs actual. The instruction documents a maximum error of `1.5 * 2^-12` which is `0.0003662109375` while scaling the `CrossPlatformMachineEpsilon` allows for an error of up to `0.000476837158`

Reply:
Just for myself, where is this instruction? In the Intel/AMD manuals?

Reply:
`rcpss` and `rsqrtss`, they would be in Volume 2 of the Intel manuals (https://software.intel.com/content/www/us/en/develop/articles/intel-sdm.html) and Volume 4 of the AMD manuals (https://developer.amd.com/resources/developer-guides-manuals/)